Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -214,10 +215,18 @@ extension BrowserViewController: WKNavigationDelegate {
                 off.compactMap { $0.rule }.forEach(controller.remove)
               
                 if let tab = tabManager[webView] {
-                    tab.userScriptManager?.isFingerprintingProtectionEnabled = domainForShields.isShieldExpected(.FpProtection)
+                    tab.userScriptManager?.isFingerprintingProtectionEnabled =
+                        !domainForShields.isAllShieldsOff()
+                        &&
+                        domainForShields.isShieldExpected(.FpProtection)
                 }
 
-                webView.configuration.preferences.javaScriptEnabled = !domainForShields.isShieldExpected(.NoScript)
+                webView.configuration.preferences.javaScriptEnabled =
+                    domainForShields.isAllShieldsOff()
+                    ||
+                    !domainForShields.isShieldExpected(.NoScript)
+                
+                print(webView.configuration.preferences.javaScriptEnabled)
             }
             
             //Cookie Blocking code below
```

Comment:
i think cookie blocking should also be disabled when shields is off?

Reply:
@diracdeltas Cookie blocking isn't on the shields panel. Cookie blocking is in settings where you have to explicitly turn on "Block All Cookies".

Panel:
![image](https://user-images.githubusercontent.com/1530031/70151544-bf9fdb80-1679-11ea-832c-19312a2fdc60.png)


Settings (block all cookies):
![image](https://user-images.githubusercontent.com/1530031/70151608-cf1f2480-1679-11ea-9e17-0f7f46e73a85.png)

In other words, Cookie Blocking isn't a shield on iOS (Desktop has it as a shield).

Reply:
ah i forgot it wasn't in shields on iOS